src/Uno.UWP/Devices/Geolocation/Geolocator.iOSmacOS.cs

```diff
@@ -38,6 +38,11 @@ private void _locationManager_LocationsUpdated(object sender, CLLocationsUpdated
 			BroadcastStatusChanged(PositionStatus.Initializing);
 		}
 
+		/// <summary>
+		/// Return an instance of CLLocationManager to set properties
+		/// </summary>
+		public CLLocationManager GetLocationManager { get { return _locationManager; } }
```

Comment:
Thanks! To avoid augmenting the public API and cause trouble in case of a refactoring of this class, let's move this to a `GeolocatorHelper` static class. The class can be located in the same folder, in the same namespace:

```csharp
using CoreLocation;

namespace Windows.Devices.Geolocation;

public static class GeolocatorHelper
{
		/// <summary>
		/// Return an instance of CLLocationManager to set properties
		/// </summary>
		public static CLLocationManager GetLocationManager(this Geolocator instance) 
           => instance.LocationManager;
}
```

You'll also need to make change like this:
```suggestion
		internal CLLocationManager LocationManager => _locationManager;
```

Reply:
Understood Thank you!